CMakeLists.txt

```diff
@@ -115,12 +115,12 @@ else()
 endif()
 
 # Enable specific certs
```

Comment:
Is it worth group the options (new lines 77-80) with this section and add a comment about intended use only for samples? (Sample Cert section starts at new line 110)

Reply:
Good point.